src/System.Runtime.InteropServices/tests/Configurations.props

```diff
@@ -2,8 +2,11 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netstandard;
-      netcoreapp;
+      netcoreapp-Windows_NT;
+      netcoreapp-Unix;
```

Comment:
Unfortunatelly there is one condition inside the csproj that does use `TargetsWindows` which means the suffixes are required on the cases where we actually want that condition to be true.